### PR TITLE
RD-1335 RD-1356 (master) Fix data fetching issues in DynamicDropdown component and in system tests

### PR DESCRIPTION
**Pull request by qooban**
With https://github.com/cloudify-cosmo/cloudify-stage/pull/1157 I added `useEffect` for `fetchUrl` prop change. It was calling `loadMore()` method. This method is changing page (offset) and was designed to handle scrolling down in the dropdown to load more data. That resulted in problems in displaying list of options in dropdowns using DynamicDropdown component. Removing it for now as we don't need it, yet (Labels in UI feature is not going to be release in 5.2). 

It will be revisited how to handle changing `fetchUrl` within [RD-1271](https://cloudifysource.atlassian.net/browse/RD-1217).

This fix solves issues in the following specs:
1. widgets/filter_spec.js
2. widgets/blueprint_actions_spec.js
3. widgets/blueprint_info_spec.js
4. widgets/blueprint_sources_spec.js
5. widgets/blueprint_spec.js
6. widgets/deployment_info_spec.js
7. widgets/nodes_stats_spec.js
8. widgets/outputs_capabilities_spec.js

Once this is merged, I'll cherry pick commit from `master` branch and create PR for `5.2-build` branch.

### CODE REVIEW

**Review by kubama — APPROVED** [review]

File: test/cypress/integration/widgets/blueprints_spec.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
 describe('Blueprints widget', () => {
     const blueprintNamePrefix = 'blueprints_test';
     const emptyBlueprintName = `${blueprintNamePrefix}_empty`;
+    const blueprintsWidgetConfiguration = { displayStyle: 'table', clickToDrillDown: true, pollingTime: 5 };
```
Comment (Gelio) [inline test/cypress/integration/widgets/blueprints_spec.js:4]: And you managed to squeeze in a small refactor :smile: Nice! I love me some deduplication

**Review by Gelio — COMMENTED** [review]
Just to be on the safe side - did you rerun all the system tests that were failing and did they pass now :smile: ?

**Comment by qooban** [comment]
@Gelio Yes, all specs listed in the description except `blueprints_spec` passed after https://github.com/cloudify-cosmo/cloudify-stage/pull/1192/commits/a7f3f474b145b7fb1930426efe9571857e5930ff . See https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/146/ for a reference.

With the https://github.com/cloudify-cosmo/cloudify-stage/pull/1192/commits/482a834bfacd30875a0267cd71de7eac175fb824 also `blueprints_spec` passed. I did not re-run all tests after that, but reproduced the problem locally and checked that after the fix it is fine.

File: test/cypress/integration/widgets/blueprints_spec.js (hunk 1 above)
Comment (qooban) [reply test/cypress/integration/widgets/blueprints_spec.js:4]: Yep 👍🏻 Thanks :-)

**Comment by Gelio** [comment]
@qooban Great :smile: `plugin_spec` was fixed by @geokala, so it looks like we only have issues with map connectivity, and topology, after all :slightly_smiling_face:

**Pull request merged**